shared-helpers/src/locales/general.json

```diff
@@ -1099,6 +1099,11 @@
   "region.name": "Local Region",
   "search.filters": "Filters",
   "search.totalResults": "Total results",
+  "session.voided.header": "Session expired",
+  "session.voided.context1": "You've been signed out and your application for ",
```

Comment:
**note:** I know we know this, but string concatenation like this isn't workable in translations, but I think we would need human translations for a long term solution